src/transformers/models/layoutlmv2/modeling_layoutlmv2.py

```diff
@@ -36,10 +36,12 @@
     QuestionAnsweringModelOutput,
     SequenceClassifierOutput,
     TokenClassifierOutput,
+    ReOutput,
 )
 from ...modeling_utils import PreTrainedModel, apply_chunking_to_forward
 from ...utils import logging
 from .configuration_layoutlmv2 import LayoutLMv2Config
+from .re import REDecoder
```

Comment:
You can include the decoder inside `modeling_layoutlmv2.py`.

Our philosophy is a single model = a single script.